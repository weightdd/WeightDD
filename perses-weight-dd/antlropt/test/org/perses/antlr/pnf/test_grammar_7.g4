grammar test_grammar_7;

start
    : ('A' ('C' | 'D')+ 'B'?)* 'E'?
    ;