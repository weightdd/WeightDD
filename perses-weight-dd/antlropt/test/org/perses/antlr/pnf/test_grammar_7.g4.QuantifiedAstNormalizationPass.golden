// Auto-generated by Perses AST.
// DO NOT MODIFY.
grammar test_grammar_7;



start
    : kleene_star__start_5 optional__start_6
    ;

aux_rule__start_1
    : 'C'
    | 'D'
    ;

kleene_plus__start_2
    : aux_rule__start_1+
    ;

optional__start_3
    : 'B'?
    ;

aux_rule__start_4
    : 'A' kleene_plus__start_2 optional__start_3
    ;

kleene_star__start_5
    : aux_rule__start_4*
    ;

optional__start_6
    : 'E'?
    ;

